packages/squiggle-lang/__tests__/Distributions/Samples_test.res

```diff
@@ -0,0 +1,46 @@
+open Jest
+open Expect
+
+let makeTest = (~only=false, str, item1, item2) =>
+  only
+    ? Only.test(str, () => expect(item1) -> toEqual(item2))
+    : test(str, () => expect(item1) -> toEqual(item2))
```

Comment:
I feel like we have this code in a few places at this point, could likely refactor it to TestUtils somewhere.